contracts/carbon/Strategies.sol

```diff
@@ -178,14 +178,22 @@ abstract contract Strategies is Initializable {
     // accumulated fees per token
     mapping(Token => uint256) internal _accumulatedFees;
 
+    // mapping between a pair id to its custom trading fee (in units of PPM)
+    mapping(uint128 pairId => uint32 fee) internal _customTradingFeePPM;
+
     // upgrade forward-compatibility storage gap
-    uint256[MAX_GAP - 4] private __gap;
+    uint256[MAX_GAP - 5] private __gap;
 
     /**
      * @dev triggered when the network fee is updated
```

Comment:
Minor: "network" is probably a copy/paste mistake from previous code.

Reply:
This is not a part of this PR.